src/mailmojo-plugin.php

```diff
@@ -76,7 +81,18 @@ public function loadTextDomain () {
 	public function getSettingsPageUrl () {
 		global $blog_id;
 		$adminUrl = get_admin_url($blog_id);
-		return "{$adminUrl}options-general.php?page={MailMojoSettings::MENU_SLUG}";
+		$slug = MailMojoSettings::MENU_SLUG;
+		return "{$adminUrl}options-general.php?page={$slug}";
+	}
+
+	/**
+	 * Return access token stored as an option if it exist.
+	 *
+	 * @return string
+	 */
+	public static function getAccessToken () {
+		$options = get_option('mailmojo_options');
+		return !empty($options['access_token']) ? $options['access_token'] : '';
```

Comment:
This is the same logic as we do when outputting the access token field in `mailmojo-settings.php`. Could this be aligned with a common accessor somehow? I'm not quite on top of how all these parts work together.